src/projects/trio/trio.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+
+#include <array>
+#include <vector>
+#include <unordered_map>
+#include <sequences/seqio.hpp>
+#include "common/logging.hpp"
+#include <common/string_utils.hpp>
+#include "lja/multi_graph.hpp"
+#include "lja/subdataset_processing.hpp"
+
+
+
+struct HaplotypeRemover {
+    multigraph::MultiGraph &mg;
+    haplo_map_type haplotypes;
+    logging::Logger &logger_;
+    char haplotype_;
+    static const size_t MAX_TIP_LENGTH = 1000000;
+    static constexpr double BULGE_MULTIPLICATIVE_CUTOFF = 1.2;
+//Bridges of wrong haplotype longer that this cutoff are deleted, shorter are saved;
+    static const size_t SAVED_BRIDGE_CUTOFF = 100000;
+    std::experimental::filesystem::path out_dir;
+    std::unordered_map<std::string, std::string> bulges;
+    HaplotypeRemover(logging::Logger &logger, multigraph::MultiGraph &mg,
+                     const std::experimental::filesystem::path &haployak, const char haplotype, const std::experimental::filesystem::path &out_dir):logger_(logger), mg(mg), haplotype_(haplotype), out_dir(out_dir) {
+
+        auto bulges = getBulgeLabels();
+        logger_.info() << "got "<< bulges.size() << " bulges\n";
+        haplo_map_type haplotypes;
+        string s;
+        std::ifstream haplo_file(haployak);
```

Comment:
What is haployak? Is it an outputof some external classification program? Can we incorporate it inside here?

Reply:
Yes, but currently I use the results of yak triobin as file. Let's have it in next PR?